src/main_app/jobs_workers/crop_main_files_worker.py

```
@@ -0,0 +1,142 @@
+        result["status"] = "failed"
+        result["error"] = str(e)
+        result["error_type"] = type(e).__name__
+
+    # Finalize timestamps
+    result["completed_at"] = datetime.now().isoformat()
+    final_status = result.get("status", "completed")
+
+    # Save final results
+    try:
+        jobs_service.save_job_result_by_name(result_file, result)
+    except Exception as exc:
```

Ruff (0.15.1)

[error] Local variable `exc` is assigned to but never used

Remove assignment to unused variable `exc`

(F841)